GTG/core/sorters.py

```diff
@@ -31,14 +31,46 @@ def unwrap(row, expected_type):
 
     return item
 
+class ReversibleSorter(Gtk.Sorter):
+
+    def __init__(self) -> None:
+        self._reverse: bool = False
+        super().__init__()
+
+
+    @property
+    def reverse(self) -> bool:
+        return self._reverse
 
-class TaskTitleSorter(Gtk.Sorter):
-    __gtype_name__ = 'TaskTitleSorter'
 
-    def __init__(self):
-        super(TaskTitleSorter, self).__init__()
+    @reverse.setter
+    def reverse(self, value: bool) -> None:
+        self._reverse = value
+        self.changed(Gtk.SorterChange.INVERTED)
 
 
+    def reversible_compare(self, first, second) -> Gtk.Ordering:
```

Comment:
An alternative is to have the equal test first:
```
        if first == second:
            return Gtk.Ordering.EQUAL

        if first < second:
            if self._reverse:
                return Gtk.Ordering.LARGER
            else:
                return Gtk.Ordering.SMALLER

        if self._reverse:
            return Gtk.Ordering.SMALLER
        else:
            return Gtk.Ordering.LARGER
```

The entire thing looks a bit clunky to me, but is apparently the GTK way to solve this problem, so LGTM.

Reply:
Ah that's a bit cleaner. Thanks! But yeah, I couldn't find a less clunky way of doing this